File: They say that only the name is long enough to attract attention. They also said that only a simple Kata will have someone to solve it. This is a sadly story #1: Are they opposite?/They say that only the name is long enough to attract attention. They also said that only a simple Kata will have someone to solve it. This is a sadly story #1: Are they opposite?.py
def is_opposite(s1,s2):
    if (s1=="" or s2==""):
        return False
    
    for i1,i2 in zip(s1,s2):
        if not(check(i1,i2)):
            return False
    return True

def check(c1,c2):
    if c1.lower()==c2.lower():
        if (c1.isupper() and c2.islower()) or (c2.isupper() and c1.islower()):
            return True
    return False

File: They say that only the name is long enough to attract attention. They also said that only a simple Kata will have someone to solve it. This is a sadly story #1: Are they opposite?/They say that only the name is long enough to attract attention. They also said that only a simple Kata will have someone to solve it. This is a sadly story #1: Are they opposite?.js
function isOpposite(s1, s2) {
  if ((!s1 && !s2) || s1.length != s2.length) return false;
  if (s1.toLowerCase() != s2.toLowerCase()) return false;
  for (let i = 0; i < s1.length; i++) {
    if (s1[i] === s2[i]) {
      return false;
    }
  }
  return true;
}
